tests/mock.c

```diff
@@ -203,22 +211,22 @@ int ioctl(int fd, unsigned long request, ...)
 				goto out_EINVAL;
 			}
 			pr->status = 0; /* return success */
-			/* TODO: reflect reconfiguration (chagne afu_id?) */
+			/* TODO: reflect reconfiguration (change afu_id?) */
 			/* generate hash for bitstream data */
-            hash = stupid_hash((uint32_t*)pr->buffer_address, pr->buffer_size / 4);
-            /* write hash to file in tmp */
-            strncpy_s(hashfilename, MAX_STRLEN, mock_devs[fd].pathname, strlen(mock_devs[fd].pathname) + 1);
-            strncat_s(hashfilename, MAX_STRLEN, HASH_SUFFIX, sizeof(HASH_SUFFIX));
-
-            FILE* hashfile = fopen(hashfilename, "w");
-            if (hashfile) {
-                fwrite(&hash, sizeof(hash), 1, hashfile);
-                fclose(hashfile);
-            }
-            retval = 0;
-            errno = 0;
-            break;
-        case FPGA_FME_PORT_ASSIGN:
+			hash = stupid_hash((uint32_t*)pr->buffer_address, pr->buffer_size / 4);
+			/* write hash to file in tmp */
+			strncpy_s(hashfilename, MAX_STRLEN, mock_devs[fd].pathname, strlen(mock_devs[fd].pathname) + 1);
+			strncat_s(hashfilename, MAX_STRLEN, HASH_SUFFIX, sizeof(HASH_SUFFIX));
+
+			FILE* hashfile = fopen(hashfilename, "w");
+			if (hashfile) {
+		                fwrite(&hash, sizeof(hash), 1, hashfile);
+				fclose(hashfile);
+			}
+			retval = 0;
+			errno = 0;
+			break;
+		case FPGA_FME_PORT_ASSIGN:
```

Comment:
Good formatting fix... 👍 
I was considering doing this when I started to look at enabling more event tests.